openvdb/tools/MeshToVolume.h

```diff
@@ -2172,6 +2175,11 @@ class VoxelizePolygons
         const ValueType dist = ValueType((voxelCenter -
             closestPointOnTriangleToPoint(prim.a, prim.c, prim.b, voxelCenter, uvw)).lengthSqr());
 
+        // Either the points may be NAN, or they could be far enough from
+        // the origin that computing distance fails.
+        if (std::isnan(dist))
```

Comment:
Would it not be preferable to use "`!std::isfinite(dist)`", to handle the case of negative infinity?